x-pack/plugins/timelines/public/components/t_grid/body/index.tsx

```diff
@@ -337,6 +341,8 @@ export const BodyComponent = React.memo<StatefulBodyProps>(
                     id={id}
                     totalItems={totalItems}
                     filterStatus={filterStatus}
+                    query={filterQuery}
+                    indexName={indexNames.join(',')}
```

Comment:
nit: Isn't the default argument of join a comma? Can't you just do `indexNames.join()`?

Reply:
Yes, you're right, the comma is the default value. I was used to be explicit with it since it is easier to understand, anyway I removed the parameter 👍